src/site/content/en/fast/codelab-serve-modern-code/index.md

```diff
@@ -38,14 +38,19 @@ optimizations:
 {% Instruction 'disable-cache', 'ol' %}
 {% Instruction 'reload-app', 'ol' %}
 
+```html
```

Comment:
I was trying to add syntax highlighting to that sniped, not that you pointed out I think it might not be necessary, do you think I should keep it as it is or remove it?

Reply:
oh. It's not meant to be a code snippet. It's an image showing the results to the reader. So the backticks should be removed. The same is true for the snippet on line 51. The backticks should be removed and the image should be allowed to display.

Reply:
I've updated the PR with the requested changes